Sources/SwifterSwift/Foundation/NSObjectExtensions

```diff
@@ -0,0 +1,24 @@
+#if canImport(Foundation) && !os(Linux)
+import Foundation
+
+// MARK: - Properties
+
+public extension NSObject{
+    /// SwifterSwift: class name.
+    class var named: String {
```

Comment:
I think `className` would be more descriptive.

Reply:
yeah,I modified